### ISSUE TITLE
Kernel freezes after running tff.federated_computation

### ISSUE THREAD

**Issue by gokulanv** [tensorflow/federated#869]
**Describe the bug**
A clear and concise description of what the bug is. It is often helpful to
provide a link to a [colab](https://colab.research.google.com/) notebook that
reproduces the bug.

**Environment (please complete the following information):** 
* OS Platform and Distribution (e.g., Linux Ubuntu 16.04): Windows
* Python package versions (e.g., TensorFlow Federated, TensorFlow): 
      tensorflow-2.2.0, tf-federated - 0.14.0
* Python version: 3.7
* Bazel version (if building from source):
* CUDA/cuDNN version: 10.1, >418.x
* What TensorFlow Federated execution stack are you using? 

Note: You can collect the Python package information by running `pip3 freeze`
from the command line and most of the other information can be collected using
TensorFlows environment capture
[script](https://github.com/tensorflow/tensorflow/blob/master/tools/tf_env_collect.sh).

**Expected behavior**
Want to set up tensorflow federated with GPU support in my local machine. Installation had no issues. But I am not run the code with tensorflow_federated

**Additional context**

_Code:_

> import collections
> import os
> import numpy as np
> import glob
> import matplotlib.pyplot as plt
> from pathlib import Path
> 
> import tensorflow as tf
> from tensorflow import reshape, nest, config
> from tensorflow.keras import losses, metrics, optimizers
> import tensorflow_federated as tff
> from tensorflow.keras.models import Sequential
> from tensorflow.keras.layers import Dense, Conv2D, Flatten, Dropout, MaxPooling2D
> 
> import nest_asyncio
> nest_asyncio.apply()
> 
> tff.federated_computation(lambda: 'hi')()
> np.random.seed(0)

After running this, I am not able to get the outputs for consecutive cells. The kernel freezes and I have it kill it everytime.

**LOGS:**

` (venv) (tensorflo-local) C:\Users\gokul\Codebase_SDD\Machine Learning\Tensorflow\federated\ToyFederatedLearning\TensorFlow\Federated>jupyter notebook
[I 23:34:06.172 NotebookApp] Serving notebooks from local directory: C:\Users\gokul\Codebase_SDD\Machine Learning\Tensorflow\federated\ToyFederatedLearning\TensorFlow\Federated
[I 23:34:06.172 NotebookApp] The Jupyter Notebook is running at:
[I 23:34:06.172 NotebookApp] http://localhost:8888/?token=3aa46721913631f4bad5ba9e8c46011fd51c777c5cd99845
[I 23:34:06.174 NotebookApp]  or http://127.0.0.1:8888/?token=3aa46721913631f4bad5ba9e8c46011fd51c777c5cd99845
[I 23:34:06.174 NotebookApp] Use Control-C to stop this server and shut down all kernels (twice to skip confirmation).
[C 23:34:06.262 NotebookApp]

    To access the notebook, open this file in a browser:
        file:///C:/Users/gokul/AppData/Roaming/jupyter/runtime/nbserver-7068-open.html
    Or copy and paste one of these URLs:
        http://localhost:8888/?token=3aa46721913631f4bad5ba9e8c46011fd51c777c5cd99845
     or http://127.0.0.1:8888/?token=3aa46721913631f4bad5ba9e8c46011fd51c777c5cd99845
[W 23:34:10.009 NotebookApp] Notebook TFFederated.ipynb is not trusted
[I 23:34:10.690 NotebookApp] Kernel started: 0ff077b0-7063-4cd9-a1a3-67175a839873
2020-06-09 23:34:26.636662: I tensorflow/stream_executor/platform/default/dso_loader.cc:44] Successfully opened dynamic library cudart64_101.dll
2020-06-09 23:34:32.733418: I tensorflow/stream_executor/platform/default/dso_loader.cc:44] Successfully opened dynamic library nvcuda.dll
2020-06-09 23:34:32.823969: I tensorflow/core/common_runtime/gpu/gpu_device.cc:1561] Found device 0 with properties:
pciBusID: 0000:01:00.0 name: GeForce GTX 1060 with Max-Q Design computeCapability: 6.1
coreClock: 1.48GHz coreCount: 10 deviceMemorySize: 6.00GiB deviceMemoryBandwidth: 178.99GiB/s
2020-06-09 23:34:32.834014: I tensorflow/stream_executor/platform/default/dso_loader.cc:44] Successfully opened dynamic library cudart64_101.dll
2020-06-09 23:34:32.842948: I tensorflow/stream_executor/platform/default/dso_loader.cc:44] Successfully opened dynamic library cublas64_10.dll
2020-06-09 23:34:32.851389: I tensorflow/stream_executor/platform/default/dso_loader.cc:44] Successfully opened dynamic library cufft64_10.dll
2020-06-09 23:34:32.857673: I tensorflow/stream_executor/platform/default/dso_loader.cc:44] Successfully opened dynamic library curand64_10.dll
2020-06-09 23:34:32.865883: I tensorflow/stream_executor/platform/default/dso_loader.cc:44] Successfully opened dynamic library cusolver64_10.dll
2020-06-09 23:34:32.873915: I tensorflow/stream_executor/platform/default/dso_loader.cc:44] Successfully opened dynamic library cusparse64_10.dll
2020-06-09 23:34:32.887805: I tensorflow/stream_executor/platform/default/dso_loader.cc:44] Successfully opened dynamic library cudnn64_7.dll
2020-06-09 23:34:32.892708: I tensorflow/core/common_runtime/gpu/gpu_device.cc:1703] Adding visible gpu devices: 0
2020-06-09 23:34:32.896278: I tensorflow/core/platform/cpu_feature_guard.cc:143] Your CPU supports instructions that this TensorFlow binary was not compiled to use: AVX2
2020-06-09 23:34:32.907407: I tensorflow/compiler/xla/service/service.cc:168] XLA service 0x23689ec41d0 initialized for platform Host (this does not guarantee that XLA will be used). Devices:
2020-06-09 23:34:32.913272: I tensorflow/compiler/xla/service/service.cc:176]   StreamExecutor device (0): Host, Default Version
2020-06-09 23:34:32.918127: I tensorflow/core/common_runtime/gpu/gpu_device.cc:1561] Found device 0 with properties:
pciBusID: 0000:01:00.0 name: GeForce GTX 1060 with Max-Q Design computeCapability: 6.1
coreClock: 1.48GHz coreCount: 10 deviceMemorySize: 6.00GiB deviceMemoryBandwidth: 178.99GiB/s
2020-06-09 23:34:32.927971: I tensorflow/stream_executor/platform/default/dso_loader.cc:44] Successfully opened dynamic library cudart64_101.dll
2020-06-09 23:34:32.933083: I tensorflow/stream_executor/platform/default/dso_loader.cc:44] Successfully opened dynamic library cublas64_10.dll
2020-06-09 23:34:32.938547: I tensorflow/stream_executor/platform/default/dso_loader.cc:44] Successfully opened dynamic library cufft64_10.dll
2020-06-09 23:34:32.943804: I tensorflow/stream_executor/platform/default/dso_loader.cc:44] Successfully opened dynamic library curand64_10.dll
2020-06-09 23:34:32.948171: I tensorflow/stream_executor/platform/default/dso_loader.cc:44] Successfully opened dynamic library cusolver64_10.dll
2020-06-09 23:34:32.953024: I tensorflow/stream_executor/platform/default/dso_loader.cc:44] Successfully opened dynamic library cusparse64_10.dll
2020-06-09 23:34:32.959809: I tensorflow/stream_executor/platform/default/dso_loader.cc:44] Successfully opened dynamic library cudnn64_7.dll
2020-06-09 23:34:32.964122: I tensorflow/core/common_runtime/gpu/gpu_device.cc:1703] Adding visible gpu devices: 0
2020-06-09 23:34:33.599328: I tensorflow/core/common_runtime/gpu/gpu_device.cc:1102] Device interconnect StreamExecutor with strength 1 edge matrix:
2020-06-09 23:34:33.604817: I tensorflow/core/common_runtime/gpu/gpu_device.cc:1108]      0
2020-06-09 23:34:33.607803: I tensorflow/core/common_runtime/gpu/gpu_device.cc:1121] 0:   N
2020-06-09 23:34:33.611780: I tensorflow/core/common_runtime/gpu/gpu_device.cc:1247] Created TensorFlow device (/job:localhost/replica:0/task:0/device:GPU:0 with 4700 MB memory) -> physical GPU (device: 0, name: GeForce GTX 1060 with Max-Q Design, pci bus id: 0000:01:00.0, compute capability: 6.1)
2020-06-09 23:34:33.624057: I tensorflow/compiler/xla/service/service.cc:168] XLA service 0x236b4085280 initialized for platform CUDA (this does not guarantee that XLA will be used). Devices:
2020-06-09 23:34:33.630275: I tensorflow/compiler/xla/service/service.cc:176]   StreamExecutor device (0): GeForce GTX 1060 with Max-Q Design, Compute Capability 6.1

`

**Comment by ambrishrawat** [comment]
Check your tornado version - https://github.com/jupyter/notebook/issues/3397

Reverting to the old version did the trick for me

**Comment by gokulanv** [comment]
Thanks @ambrishrawat 
Had fixed it already. Referred the same link.

This can be fixed by downgrading the jupyter notebook and tornado version.

Default versions of notebook >= 6.x and tornado >= 6.0

By running the command,

pip install jupyter notebook==5.7.8 tornado==4.5.0

This issue is fixed
